JoshuaKodhe/Store-Manager-API
# #161363243 Deploy to heroku

#### What does this PR do?

Have all endpoints for hosted on heroku

#### Description of Task to be completed?

Have the following endpoints working

 Endpoint | Functionality
 --- | ---
GET /products | Fetch all products
GET /products/<productId> | Fetch a single product record
GET /sales| /questions | Fetch all sale records
GET /sales/<saleId> | Fetch a single sale record
POST /products | Create a product
POST /sales | Create a sale order
POST /register| Create a user
POST /login | Login a user

#### How should this be manually tested?

Navigate to `https://a-store-manager-api.herokuapp.com/`
Copy the Url
Make sure you have an internet connection 
Append the different endpoints 
Test endpoints in postman

#### What are the relevant pivotal tracker stories?
[#161363243]
